crates/re_ui/src/toggle_switch.rs

```diff
@@ -1,12 +1,12 @@
 //! Adapted from `egui_demo_lib/src/demo/toggle_switch.rs`
 
-fn toggle_switch_ui(ui: &mut egui::Ui, on: &mut bool) -> egui::Response {
-    let interactive_size = egui::vec2(12.0, ui.spacing().interact_size.y);
-    let (interact_rect, mut response) =
-        ui.allocate_exact_size(interactive_size, egui::Sense::click());
-    let visual_size = egui::vec2(12.0, 8.0); // 12x7 in figma, but 12x8 looks _much_ better in epaint
-    let visual_rect =
-        egui::Align2::CENTER_CENTER.align_size_within_rect(visual_size, interact_rect);
+fn toggle_switch_ui(ui: &mut egui::Ui, height: f32, on: &mut bool) -> egui::Response {
```

Comment:
shouldn't all our toggle switches have the same height, everywhere?

Reply:
We don't use it at all in rerun yet, but at least in `re_ui_example` I used it twice in contexts that very specifically call for a different size:

<img width="331" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/d22c290d-7128-4925-92ec-2c4ee91e44c8">
<img width="322" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/95bad322-cb1c-4cf6-a234-d4d0d7436e80">